spec/requests/api/v1/users/sign_in_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+RSpec.describe 'User login', type: :request do
+  subject { post api_v1_user_session_path, params: user_params, as: :json }
+  before { subject }
```

Comment:
NIT:
Not sure there is a rule for this, but I always like leaving an empty line in between these

Reply:
Thanks for pointing it out!